lib/stages/build.sh

```diff
@@ -9,17 +9,17 @@ _stage_build() {
 
     # Install all dependencies
     cd ${DRUPAL_TRAVIS_DRUPAL_INSTALLATION_DIRECTORY}
-    COMPOSER_MEMORY_LIMIT=-1 composer install
+    COMPOSER_MEMORY_LIMIT=-1 composer update
 
     local installed_version=$(composer show 'drupal/core' | grep 'versions' | grep -o -E '[^ ]+$')
     local major_version="$(cut -d'.' -f1 <<<"${installed_version}")"
     local minor_version="$(cut -d'.' -f2 <<<"${installed_version}")"
 
     # When we are having Drupal 9 or if we are on at least Drupal 8.8, use core scaffold. Otherwise use the legacy scaffolding.
     if [[ "${major_version}" -gt 8 ]] || [[ "${minor_version}" -gt 7 ]]; then
-        composer require drupal/core-composer-scaffold
+        COMPOSER_MEMORY_LIMIT=-1 composer require drupal/core-composer-scaffold
```

Comment:
Instead of repeating the COMPOSER_MEMORY_LIMIT, we could add it to the configuration.sh like this: 
`export COMPOSER_MEMORY_LIMIT=${COMPOSER_MEMORY_LIMIT:-"-1"}`
That way we can explicitly overwrite the value under certain situations, and have it defaulted to -1.
It might be the case, that it is already set in the travis environment, then this would fail, but I believe it is not.